doc/configuration.rst

```diff
@@ -2006,6 +2006,19 @@ Advanced processing configuration
 
         confluence_file_suffix = '.conf'
 
+.. _confluence_table_width:
+
+.. confval:: confluence_table_width
```

Comment:
Can we rename this to `confluence_default_table_width` to be somewhat similar to the existing [`confluence_default_alignment`](https://sphinxcontrib-confluencebuilder.readthedocs.io/configuration/#confval-confluence_default_alignment) configuration? We could probably also populate this under the "Generic configuration" category (beside `confluence_default_alignment`) since this is not that much of an advanced setting in the eyes of users.

Reply:
Done.